src/pages/Home.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import ReactTypingEffect from 'react-typing-effect';
+
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+
+class Home extends Component {
+   render() {
+      return (
+         <div>
+            <Header />
```

Comment:
We can probably render the Header and Footer in App.js instead since it'll probably be common for most, if not all, pages

Reply:
Got it deleted from the Home page and added to the App.js